filter_plugins/filters.py

```diff
@@ -128,7 +128,7 @@ def _interface_check(context, interface, interface_type=None):
     return _pass()
 
 
-@jinja2.contextfilter
+@jinja2.pass_context
```

Comment:
This breaks on versions of Jinja2 older than 3.0.0. What I've done elsewhere is this:

```
# NOTE: jinja2 3.1.0 dropped contextfilter in favour of pass_context.
try:
    from jinja2 import pass_context
except ImportError:
    from jinja2 import contextfilter as pass_context
```

Reply:
That is something I am also in favor of, thanks!